cypress/support/step-definitions/common-steps.js

```diff
@@ -53,6 +57,18 @@ Given('I open deprecated {string} component page', (component) => {
   visitComponentUrl(component, 'classic', false, 'deprecated-');
 });
 
+Given('I open {string} textbox based component page in iframe', (component) => {
+  visitComponentUrl(component, 'textbox_based', true);
+});
+
+Given('I open {string} textbox based classic component page in iframe', (component) => {
+  visitComponentUrl(component, 'textbox_based_classic', true);
+});
+
+Given('I open {string} basic classic component page in iframe', (component) => {
```

Comment:
put this step under the `Given('I open {string} component page basic in iframe', (component) => {`
to make it more readable.

Reply:
I did it in this way because 'basic classic' is the name of the element in this component, so when we type the string it will create a whole name of the component

Reply:
I mean literally upper :)

Reply:
fixed